spec/spec.md

```diff
@@ -1505,24 +1510,65 @@ composed and embedded as follows:
   ****MUST**** be an array of _Input Descriptor Mapping Objects_, each being
   composed as follows:
     - The object ****MUST**** include an `id` property, and its value
       ****MUST**** be a string matching the `id` property of the _Input
       Descriptor_ in the _Presentation Definition_ the submission is related to.
-    - The object ****MUST**** include a `path` property, and its value
-      ****MUST**** be a [JSONPath](https://goessner.net/articles/JsonPath/)
+    - The object ****MUST**** include a `format` property when no  `steps` property 
+      is present, and its value ****MUST**** be a string value matching one of the 
+      [Credential Format Designation](#credential-format-designations) (`jwt`, 
+      `jwt_vc`, `jwt_vp`, `ldp_vc`, `ldp_vp`, `ldp`), to denote what data format the 
+      credential is being submitted in. If the `format` and `steps` properties are both 
+      present, produce an error and fail the submission.
+    - The object ****MUST**** include a `path` property when no  `steps` property 
+      is present, and its value ****MUST**** be a [JSONPath](https://goessner.net/articles/JsonPath/)
       string expression that selects the credential to be submit in relation
       to the identified _Input Descriptor_ identified, when executed against
       the top-level of the object the _Presentation Submission_ is embedded
-      within.
-    - The object ****MUST**** include a `format` property, and its value 
-      ****MUST**** be a string value matching one of the 
-      [Credential Format Designation](#credential-format-designations) (`jwt`, 
-      `jwt_vc`, `jwt_vp`, `ldp_vc`, `ldp_vp`, `ldp`), to denote what data format the credential is being 
-      submitted in.
+      within. If the `path` and `steps` properties are both present, produce an error 
+      and fail the submission.
+    - The object ****MAY**** include a `steps` property, and its value ****MUST**** 
+      be an array of Nested Submission Traversal Objects, composed as follows:
+        - The object ****MUST**** include a `path` property, and its value 
+          ****MUST**** be a [JSONPath](https://goessner.net/articles/JsonPath/)
+          string expression that selects a value matching a valid 
+          [Credential Format Designation](#credential-format-designations) 
+          (`jwt`, `jwt_vc`, `jwt_vp`, `ldp_vc`, `ldp_vp`, `ldp`) when executed against
+          the Current Traversal Object.
+        - The object ****MUST**** include a `format` property, and its value 
+          ****MUST**** be a string value matching one of the 
+          [Credential Format Designation](#credential-format-designations) (`jwt`, 
+          `jwt_vc`, `jwt_vp`, `ldp_vc`, `ldp_vp`, `ldp`), to denote what data format the 
+          value is that was selected via execution of the `path` property's 
+          [JSONPath](https://goessner.net/articles/JsonPath/) string expression.
+
+
+### Processing of `steps` Entries
```

Comment:
please update json schemas to represent steps. also unclear why we need steps

Reply:
``````
{
  "presentation_submission": {
    "descriptor_map": [
      { 
        "id": "banking_input_2",
        "format": "jwt_vp",
        "path": "$.outer.[0]"
        "path_nested": {
            "format": "ldp_vc",
            "path": "$.inner.[1]",
            "path_nested": {
                "format": "jwt_vc",
                "path": "$.superinner.[2]"
            }
        }
    }
  ]
}
```

alternative suggestion that mirrors `from` and `from_nested` from submission. also reflects nesting structure